megago/megago.py

```diff
@@ -246,9 +348,11 @@ def run_comparison(in_file):
 
         if len(current_ids) > 0:
             if platform.system() == "Linux":
-                p = multiprocessing.Process(target=run_process, args=(current_ids, GO_list1, GO_list2, freq_dict, queue, godag))
+                p = multiprocessing\
```

Comment:
This is more a personal opinion of me, but I'm not a fan of the `\` to split code over multiple lines. Maybe it's better if we write:

```python
p = multiprocessing.Process(
   target=run_process,
   args=(current_ids, go_list1, go_list2, freq_dict, queue, go_dag)
)
```